manifests/admin.pp

```diff
@@ -62,6 +62,8 @@
   Boolean          $set_polkit_admin_group    = true
 ){
 
+  simplib::assert_metadata( $module_name )
```

Comment:
The reintroduction of these functions at least merits a Y bump. This change may cause unexpected failures when using a version of this module greater than `~> 1`, even if the platform is unsupported by SIMP.